test/SpacesTest.jl

```diff
@@ -341,4 +341,16 @@ using ApproxFun, Compat.Test
         S = ApproxFun.PointSpace([1.0,2.0])
         @test ApproxFun.blockbandinds(FiniteOperator([1 2; 3 4],S,S)) == (0,0)
     end
+    
+    #SumSpace Conversion
+    @testset "SumSpace Conversion" begin
+        H = ApproxFun.HeavisideSpace([-1.0,0.0,1.0])
+        C = ApproxFun.ContinuousSpace(ApproxFun.PiecewiseSegment([-1.0,0,1]))
+        S = H + C
+        P = Ultraspherical(1,-1.0..0.0) ∪ Ultraspherical(1,0.0..1.0)
+        @test Conversion(S,P)[1,1]==1.0
+        @test Conversion(S,P)[1,2]==0.5
+        @test rangespace(Conversion(S,P))==Ultraspherical(1,-1.0..0.0) ∪ Ultraspherical(1,0.0..1.0)
+        @test domainspace(Conversion(S,P))==ApproxFun.SumSpace(ApproxFun.HeavisideSpace([-1.0,0.0,1.0]),ApproxFun.ContinuousSpace(ApproxFun.PiecewiseSegment([-1.0,0,1])))
+    end
```

Comment:
Add a test that evaluation is consistent:
```julia
f = Fun(S, randn(dimension(S)))
@test f(0.1) ≈ Fun(f, P)(0.1)
```

Reply:
dimension of S is infinity and didn't work, so I changed it a bit

Reply:
Ah good point. I’d just do `randn(100) ` .